test/java/seedu/addressbook/commands/FindCommandTest.java

```diff
@@ -17,13 +17,13 @@
 
 public class FindCommandTest {
 
-    private final AddressBook addressBook = new TypicalPersons().getTypicalAddressBook();
-    private final TypicalPersons td = new TypicalPersons();
+    private final AddressBook SAMPLE_ADDRESS_BOOK = new TypicalPersons().getTypicalAddressBook();
```

Comment:
Refactoring to rename this variable to full caps may not be needed as it is not a static final variable.